components/colors/index.css

```diff
@@ -1,8 +1,8 @@
 /* Basic */
 
 :root {
-  --white: #ffffff;
-  --black: #000000;
+  --white: #fff;
+  --black: #000;
```

Comment:
I find it weird when `#369` and `#336698` colors look different in my code. I like it when all HEX colors are six-digit. You can always make them shorter when optimizing the output.

Happy to complain about it later to let the PR be quickly merged.

Reply:
Good point! Fixed via 68b1f7d, ensuring we consistently use the long notation.